app/src/main/java/ch/heigvd/pro/b04/android/Question/QuestionUtils.java

```diff
@@ -14,13 +16,41 @@
 import retrofit2.Response;
 
 public class QuestionUtils {
-    private static MutableLiveData<List<Question>> localQuestions =
-            new MutableLiveData<>(new LinkedList<>());
+    private static QuestionUtils instance;
+    private static int DELAY_IN_MILLIS = 1000;
+
+    private MutableLiveData<List<Question>> localQuestions = new MutableLiveData<>(new LinkedList<>());
+    private Poll poll;
+    private String token;
+
+    private QuestionUtils(Poll poll, String token) {
+        setup(poll, token);
+    }
 
     private QuestionUtils() {
     }
 
-    private static Callback<List<Question>> callbackQuestions = new Callback<List<Question>>() {
+    private void setup(Poll poll, String token) {
+        this.poll = poll;
+        this.token = token;
+
+        setupRepeatedHandler();
+    }
+
+    private void setupRepeatedHandler() {
+        Handler handler = new Handler();
+
+        handler.postDelayed(new Runnable(){
+            public void run(){
+                Rockin.api()
+                        .getQuestions(poll.getIdModerator(), poll.getIdPoll(), token)
+                        .enqueue(callbackQuestions);
+                handler.postDelayed(this, DELAY_IN_MILLIS);
+            }
+        }, DELAY_IN_MILLIS);
+    }
```

Comment:
A bit ugly tbh. I think you should add an issue so we can improve this and fix it later.

Reply:
I think that this is the android way for small intervals. Have you seen anything else done ?

https://developer.android.com/reference/android/os/Handler

Reply:
It's more the fact that it's static that is not awesome :D